Index: README.md
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># Foodsy\nFoodsy\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/README.md b/README.md
--- a/README.md	(revision adf31bd598fb3ef8a06ec0f84f9b5254ace479fb)
+++ b/README.md	(date 1747757274695)
@@ -1,2 +1,3 @@
 # Foodsy
-Foodsy
+- [ ] Foodsy
+
